### DISCUSSION TITLE
3rd Party Dependencies/Libraries (Installation Issue)

### DISCUSSION THREAD

**Discussion by extrasauce69** [inspircd/inspircd#2081] [Support]
I am installing inspircd 3 from source on Ubuntu 22. I am new to linux. I got it up and running, but when I tried adding SSL, I realized I had to start over. I reinstalled Ubuntu and getting some build issues with inspircd.

I downloaded the source files and extracted.
I ran "perl ./configure" and enabled all modules
When I "make" I get this error:
"argon2.h no such file or directory"
It looks like it is looking for some files for argon2 module.
I ran "sudo apt install argon2" and reran "make" but this didn't help.
Where can I find these dependencies? Is this documented somewhere? I looked everywhere.
Thanks!

**Comment by sadiepowell** [comment]
Debian-based distros split out their development packages from their binary packages. You need to install the development header package for Argon2 `libargon2-dev`. For some packages you may also need pkg-config.

**Reply by extrasauce69** [reply]
thank you!
